README.rst

```diff
@@ -133,74 +89,22 @@ Taisei stores all data in a platform-specific directory:
-
-There are a few ways to generate a custom mapping:
-
--  You can use the
-   `controllermap <https://aur.archlinux.org/packages/controllermap>`__ utility,
-   which `comes with SDL source code
-   <https://hg.libsdl.org/SDL/file/68a767ae3a88/test/controllermap.c>`__.
--  If you use Steam, you can configure your controller there. Then you can add
-   Taisei as a non-Steam game; run it from Steam and everything should *just
-   work™*. In case you don't want to do that, find ``config/config.vdf`` in your
-   Steam installation directory, and look for the ``SDL_GamepadBind`` variable.
-   It contains a list of SDL mappings separated by line breaks.
--  You can also try the `SDL2 Gamepad Tool by General Arcade
-   <http://www.generalarcade.com/gamepadtool/>`__. This program is free to use,
-   but not open source.
--  Finally, you can try to write a mapping by hand. You will probably have to
-   refer to the SDL documentation. See `gamecontrollerdb.txt
-   <misc/gamecontrollerdb/gamecontrollerdb.txt>`__ for some more examples.
-
-Once you have your mapping, there are two ways to make Taisei use it:
-
--  Create a file named ``gamecontrollerdb.txt`` where your config, replays and
-   screenshots are, and put each mapping on a new line.
--  Put your mappings in the environment variable ``SDL_GAMECONTROLLERCONFIG``,
-   also separated by line breaks. Other games that use the GameController API
-   will also pick them up.
-
-When you're done, please consider contributing your mappings to
-`SDL <https://libsdl.org/>`__,
-`SDL_GameControllerDB <https://github.com/gabomdq/SDL_GameControllerDB>`__,
-and `us <https://github.com/taisei-project/SDL_GameControllerDB>`__, so
-that other people can benefit from your work.
-
-Also note that we currently only handle input from analog axes and digital
-buttons. Hats, analog buttons, and anything more exotic will not work, unless
-remapped.
 
 Troubleshooting
 ---------------
 
-Sound problems (Linux)
-^^^^^^^^^^^^^^^^^^^^^^
-
-If your sound becomes glitchy, and you encounter lot of console messages like:
+If you're having issues with low framerates, sound playback issues, or gamepad
+support, please see the `Troubleshooting Guide <doc/TROUBLESHOOTING.rst>`__.
 
-::
+If you find any other bug not covered in that, feel free to
+`open up an issue <https://github.com/taisei-project/taisei/issues>`__. If you
+do, please give as much information as you can, such as:
 
-    ALSA lib pcm.c:7234:(snd_pcm_recover) underrun occurred
+    1. OS version (i.e: Windows 10, macOS 10.16, Ubuntu Linux 19.10)
+    2. Hardware (i.e: Intel processor, AMD/NVIDIA graphics card, etc)
+    3. Specific problem you're experiencing
+    4. Logs, screenshots, or how to reproduce the the issue
```

Comment:
I like the idea, but it's probably better implemented as a github issue template. I don't think it's possible to submit one in a pull request, but I can add it after this lands.

Reply:
Sure, sounds good!